libs/domain/order/confirmation-banner/confirmation-banner.styles.ts

```diff
@@ -5,25 +5,20 @@ export const styles = css`
     display: flex;
     flex-direction: column;
     align-items: center;
-    line-height: 24px;
     background-color: var(--oryx-color-primary-3);
     padding-block-end: 48px;
   }
 
-  oryx-heading {
+  section {
     display: flex;
     align-items: center;
     color: var(--oryx-color-primary-9);
     gap: 40px;
     margin: 40px 0 16px;
   }
 
-  h1 {
-    font-size: 26px;
-    line-height: 38px;
-  }
-
   p {
     margin: 0;
+    line-height: 24px;
```

Comment:
wondering if this can be 22px to stay consistent with standard line height

Reply:
This is according the designs, but agree, has no so much sense

Reply:
Let's drop it and add a note to the design please. If we get push back, which I don't think, we'll fix it going forward

Reply:
Oh, the same thing, SF designs uses BO tokens, dropped